README.md

```diff
@@ -8,24 +8,16 @@ gprbuild and link the resulting library.
 
 ## Dependencies
 
-To build this project additionally to Rust an Ada toolchain is needed.
-Furthermore the following libraries are required:
+To build this project a working installation of [alire](https://alire.ada.dev/) is
+required. The build script will automatically download the required dependencies.
+The toolchain and dependencies used to build this crate are only available for
+the crates build process. For any Ada projects that are being built with this
```

Comment:
Should we document the `alr build` process if it doesn't build?

Reply:
I think you should state that one must copy the build script from the example/hello project.

Also this line seems to be unecessary https://github.com/jklmnn/gpr-rust/blob/8414ee144f17401cc38e75abb24889336f29b705/examples/ada_hello/build.rs#L1

Reply:
I'm not sure if this paragraph transports the right message. It is supposed to refer to the build script of gpr-rust itself. Maybe I should remove

> The build script will automatically download the required dependencies.
The toolchain and dependencies used to build this crate are only available for
the crates build process.

altogether as it can be misleading. This information only makes sense if you were used to it not being able to build on its own. For any new user it might be just confusing.

> Also this line seems to be unecessary when building your own project:

I just checked, it would not be needed if I use `gpr::Project::load` but the example uses only `Project::load` so I have to `use` it.

Reply:
Yes if you don't have the history of the project it makes sense to remove it.